src/continuum_opacity/opacity_H.jl

```diff
@@ -101,26 +131,23 @@ Eqn 8.12 of Grey (2005) implicitly assumes that (U(H⁻,T)/ U(H I,T)) is always
 This expression is simplicitly a rewritten form of: αff_H⁻ * n(H⁻)/n(H I) where n(H⁻)/n(H I) has
 been replaced with the expanded form of the saha equation, in which n₀ = n(H⁻) and n₁ = n(H I).
 
-Combining 8.18, and 8.19 of Gray (2005), indicate that the version opacity contribution of H⁻
-bound-free absorption (with stimulated emission correction) is given by:
+Combining 8.18, and 8.19 of Gray (2005), indicate that the opacity contribution of H⁻ bound-free
+absorption (with stimulated emission correction) is given by:
 
-                      n(H⁻)                                  n(H I)          n(H I) + n(H II)
-    κ_ν = α_bf(H⁻) * ------  * (1 - exp(-h*ν/(k*T))) * ------------------ * -----------------
-                     n(H I)                             n(H I) + n(H II)            ρ
+``\\kappa_\\nu = \\alpha_{bf}(H^-) \\frac{n(H^-)}{n(H I)} (1 - \\exp \\left( \\frac{-h\\nu}{k T}\\right) \\frac{n(H I)}{n(H I) + n(H II)} \\frac{n(H I) + n(H II)}{\\rho}``
 
-This can be rewritten as: κ_ν = α_bf(H⁻) * n(H⁻) * (1 - exp(-h*ν/(k*T))) / ρ
+This can be rewritten as: ``\\kappa_\\nu = \\alpha_{bf}(H^-) n(H⁻) (1 - \\exp \\left( \\frac{-h\\nu}{k T}\\right)  / \\rho``
 
-This function uses the polynomial provided in equation 8.11 of Gray (2005), that fits the tabulated
-data from Wishart (1979). While Gray (2005) claims that the polynomial fits the data with 0.2%
-precision for 2250 Å ≤ λ ≤ 15000 Å, in practice we find that it fits the data to better than 0.25%
-precision. Wishart (1979) expects the tabulated data to have better than 1% percent accuracy.
+Wishart (1979) expects the tabulated data to have better than 1% percent accuracy. Mathisen (1984)
+suggests that this data has better than 3% accuracy.
 """
 function Hminus_bf(nH_I_div_partition::Real, ne::Real, ν::Real, ρ::Real, T::Real, 
                    ion_energy_H⁻::Real = 0.7552)
-    αbf_H⁻ = _Hminus_bf_cross_section(ν) # does not include contributions from stimulated emission
-    stimulated_emission_correction = (1 - exp(-hplanck_cgs*ν/(kboltz_cgs*T)))
-    n_H⁻ = _ndens_Hminus(nH_I_div_partition, ne, T, _H⁻_ion_energy)
-    αbf_H⁻ * n_H⁻ * stimulated_emission_correction / ρ
+    λ = c_cgs*1e8/ν # in ångstroms
+    tmp = _Hminus_bf_cross_section(λ, ion_energy_H⁻) # in units of megabarn
+    # convert from megabarn to cm² and include contributions from stimulated emission  1e-18
+    αbf_H⁻ = (1 - exp(-hplanck_cgs*ν/(kboltz_cgs*T))) * tmp * 1e-18
+    _ndens_Hminus(nH_I_div_partition, ne, T, _H⁻_ion_energy) * αbf_H⁻ / ρ
```

Comment:
`αbf_H⁻` isn't really an alpha value is it?

```suggestion
    cross_section = _Hminus_bf_cross_section(λ, ion_energy_H⁻) # in units of megabarn
    # convert from megabarn to cm² and include contributions from stimulated emission  1e-18
    cross_section *= (1 - exp(-hplanck_cgs*ν/(kboltz_cgs*T))) * 1e-18
    _ndens_Hminus(nH_I_div_partition, ne, T, _H⁻_ion_energy) * cross_section / ρ
```

Reply:
This is the Gray (2005) notation. They use α to refer to the cross-section (unlike Rybicki & Lightman who I think use σ?). I'll definitely make this change, but it's something to keep in mind documentation-wise.

Reply:
Ah yes, sorry I keep forgetting that.